deploy/nstemplatetiers/base1ns-gotemplate/cluster.yaml

```diff
@@ -0,0 +1,268 @@
+apiVersion: toolchain.dev.openshift.com/v1alpha1
+kind: TierTemplate
+metadata:
+  name: ttr-clusterresources-go-template
+spec:
+  template:
+    apiVersion: template.openshift.io/v1
+    kind: Template
+    metadata: {}
+    objects:
+    - apiVersion: quota.openshift.io/v1
+      kind: ClusterResourceQuota
+      metadata:
+        name: for-${SPACE_NAME}-deployments
+      spec:
```

Comment:
As discussed during our call, let's simplify this part

Reply:
updated it